salt/cli/caller.py

```diff
@@ -51,10 +51,10 @@ def factory(opts, **kwargs):
             ttype = opts["pillar"]["master"]["transport"]
 
         # switch on available ttypes
-        if ttype in ("zeromq", "tcp", "detect"):
+        if ttype in ("zeromq", "tcp", "rabbitmq", "detect"):
```

Comment:
**question** is this the right place for this? I'm not super familiar with the transport layer, but it seems weird to have a ZeroMQCaller for rabbitmq. Or is it just poor/out-of-date naming? :thinking: